host-configs/LLNL/ruby-clang-14.cmake

```diff
@@ -2,5 +2,7 @@ include(${CMAKE_CURRENT_LIST_DIR}/../../src/coreComponents/LvArray/host-configs/
 
 # MPI
 set(MPI_HOME /usr/tce/packages/mvapich2/mvapich2-2.3.7-clang-14.0.6-magic CACHE PATH "")
+#set( CMAKE_CXX_FLAGS "-march=x86-64-v2 -mno-avx512f" CACHE STRING "" FORCE)
```

Comment:
Do we want to keep these options commented out? If so maybe add a comment saying why they're there.

Reply:
I can add a comment. Someone is going to have to come back and investigate.